### PR TITLE
Added support for Pegasus Powerbox Advance to the existing PPB driver

### PR DESCRIPTION
**Pull request by aaronfreimark**
I made changes as requested. Now, instead of a new driver, I extended the aux_ppb driver to detect both standard and "Advance" versions of the Pegasus Powerbox. I have tested this with my own PPBA, but I do not have a PPB to test with. 

Feedback is more than welcome!

### CODE REVIEW

File: indigo_drivers/aux_ppb/indigo_aux_ppb.c [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 // version history
 // 2.0 by Peter Polakovic <peter.polakovic@cloudmakers.eu>
+// 2.0.1 Added support for Pocket Powerbox Advance
```
Comment (rumengb) [inline indigo_drivers/aux_ppb/indigo_aux_ppb.c:21]: this is not version 2.0.1 you can just add date your name and what is changed

however you should increase the driver version 
#define DRIVER_VERSION 0x0011 -> #define DRIVER_VERSION 0x0012

**Comment by aaronfreimark** [comment]
I pushed the changes recommended by @rumengb. Thank you for the feedback -- it's good for me to improve. Waiting on additional comments by @polakovic now!

File: indigo_drivers/aux_ppb/indigo_aux_ppb.c [hunk 2]
```diff
@@ -130,7 +137,14 @@ static indigo_result aux_attach(indigo_device *device) {
 		if (AUX_POWER_OUTLET_PROPERTY == NULL)
 			return INDIGO_FAILED;
 		indigo_init_switch_item(AUX_POWER_OUTLET_1_ITEM, AUX_POWER_OUTLET_1_ITEM_NAME, "Power outlets", true);
-		indigo_init_switch_item(AUX_POWER_OUTLET_2_ITEM, AUX_POWER_OUTLET_2_ITEM_NAME, "DSLR outlet", true);
+		indigo_init_switch_item(AUX_POWER_OUTLET_2_ITEM, AUX_POWER_OUTLET_2_ITEM_NAME, "DSLR outlet", true); // In PPBA this can be 0 or 1, but can ALSO be set to variable power 3, 5, 8, 9 or 12 [Volts].
+
+		// PPBA ONLY
+		AUX_POWER_OUTLET_STATE_PROPERTY = indigo_init_light_property(NULL, device->name, AUX_POWER_OUTLET_STATE_PROPERTY_NAME, AUX_GROUP, "Power outlets state", INDIGO_OK_STATE, 4);
```
Comment (rumengb) [inline indigo_drivers/aux_ppb/indigo_aux_ppb.c:143]: I saw someting more: you initialize AUX_POWER_OUTLET_STATE_PROPERTY with  4 items, but you initialize only 1 item. Am I missing something?

Comment (aaronfreimark) [reply indigo_drivers/aux_ppb/indigo_aux_ppb.c]: Ah, THAT's what that number is for... I needed to read the spec more closely.

**Comment by polakovic** [comment]
I fixed a some bugs and did some cleanup in my own code as well, pls. can you test it if you have physical hardware? Thank you!

**Pull request merged**